Protactor/conf.js

```diff
@@ -22,10 +22,12 @@ exports.config = {
     baseUrl: 'https://www.saucedemo.com',
     jasmineNodeOpts: {
         timeout: 1000 * 60 * 5,
+        defaultTimeoutInterval: 180000,
     },
-    onComplete(passed) {
-        return browser.getSession().then(
-            session => saucelabs.updateJob(session.getId(), { passed }),
-        )
-    },
+
+    onComplete() {
+        browser.getSession().then((session) => {
+            console.log('SauceOnDemandSessionID=' + session.getId() + ' job-name=' + capabilities.name);
```

Comment:
Let's keep this as is. This way customer have to use the Sauce CI Plugin to update job. Instead we should rather make use of our Sauce Node package.